sql/rpl_rli.cc

```diff
@@ -1345,7 +1350,6 @@ void Relay_log_info::cleanup_context(THD *thd, bool error) {
     };);
   }
   m_table_map.clear_tables();
```

Comment:
Wouldn't it be good to move as well for better code readability?

Reply:
I'm not sure that understand right, could you explain please: move `m_table_map.clear_tables();` before `slave_close_thread_tables(thd);`? Yes, but probably it requires to rerun all replication tests again.

Reply:
Done